src/tests/test_plot.py

```diff
@@ -146,3 +146,19 @@ def test_internal_histogram_no_nulls(tmp_empty, ip):
         "%sqlplot histogram --table data.csv --column age --table data.csv"
     )
     assert isinstance(out.result, matplotlib.axes._axes.Axes)
+
+
+def test_histogram_density(tmp_empty, ip):
+    Path("data.csv").write_text(
+        "name,age\nDan,33\nBob,19\nSheri,45\nVin,33\nMick,38\nJay,33\nSky,33"
+    )
+    ip.run_cell("%sql duckdb://")
+    ip.run_cell(
+        """%%sql --save test_dataset --no-execute
+SELECT *
+FROM data.csv
+"""
+    )
+    cmd = "%sqlplot histogram --table data.csv --column age --density"
```

Comment:
Please add tests for both `--density` and `-d` using parametrize

Reply:
When parametrizing, I end up getting the output add up to 2 instead of 1, since there's some reuse of the fixture behind the scenes.

```python
def test_histogram_density(tmp_empty, ip, param):
    Path("data.csv").write_text(
        "name,age\nDan,33\nBob,19\nSheri,45\nVin,33\nMick,38\nJay,33\nSky,33"
    )
    ip.run_cell("%sql duckdb://")
    ip.run_cell(
        """%%sql --save test_dataset --no-execute
SELECT *
FROM data.csv
"""
    )
    cmd = f"%sqlplot histogram {param} --table data.csv --column age"
    out = ip.run_cell(cmd)
    assert np.isclose(sum(p.get_height() for p in out.result.patches), 1)
```

Any advice on how to clean out the output? Seems like there must be some way to reset the ip fixture?

Reply:
Also not sure how the comparison tests reference what is being compared, can you point me in the right direction?

Reply:
you might try by switching `ip` -> `ip_empty`, but I don't think there's any state that would affect the plot. the definition of both fixtures is in `conftest.py` so you might wanna take a look there